plugin/src/main/scala/io/github/davidgregory084/TpolecatPlugin.scala

```diff
@@ -17,38 +17,39 @@
 package io.github.davidgregory084
 
 import sbt.Keys._
-import sbt._
```

Comment:
This is a bit unfortunate, oh well :)

Reply:
Yeah, I was not aware that there was a `ScalaVersion` in `sbt._`, that namespace is getting very busy now